src/icons/ErrorCircleIndicatorIcon.tsx

```diff
@@ -0,0 +1,13 @@
+import { createSvgIcon } from "@material-ui/core/utils";
+import React from "react";
+
+export const ErrorCircleIndicatorIcon = createSvgIcon(
+  <>
+    <circle cx="12" cy="12" r="12" fill="#FEADAD" />
+    <circle cx="12" cy="12" r="9" fill="white" />
+  </>,
+  "ErrorCircleIndicatorIcon"
+);
+(ErrorCircleIndicatorIcon as React.ComponentType).defaultProps = {
```

Comment:
Why not use typescript interface for `viewBox` instead of `defaultProps`? I see that it's used in some icons so it can be reused.